stubs/openpyxl/openpyxl/cell/rich_text.pyi

```diff
@@ -0,0 +1,24 @@
+from typing import Protocol, overload
+from typing_extensions import Self
+
+from openpyxl.descriptors import Strict, String, Typed
+
+class _HasStr(Protocol):
```

Comment:
As `object` defines `__str__`, I'm not sure it's possible to come up with a class that _doesn't_ implement this protocol from a type checker's point of view

Reply:
We could just use `object` here instead. But I'd at least alias `_HasStr` to `object` to make the intention clear. (Even if the implementation checking for `__str__` is nonsense.)

Reply:
> But I'd at least alias `_HasStr` to `object` to make the intention clear.

I'd be much happier with that — it would be more obvious that we knew what we were doing, rather than it "looking like an obvious mistake" :)

Though `list[object]` in a parameter annotation will be pretty problematic due to invariance, so `Any` might be better